CONTRIBUTING.md

```diff
@@ -26,6 +26,16 @@ Before contributing, you will probably have to create a RFC on this [strapi/rfcs
 
 This project and everyone participating in it are governed by the [Strapi Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please read the [full text](CODE_OF_CONDUCT.md) so that you can read which actions may or may not be tolerated.
 
+## Contributor License Agreement (CLA)
+
+### Individual
+
+In order to accept your pull request, we need you to submit a CLA. You only need to do this once. If you are submitting a pull request for the first time, you can complete your CLA [here](https://cla.strapi.io/strapi/strapi) or just submit a Pull Request and our CLA Bot will ask you to sign the CLA before merging your Pull Request.
+
+### Company
+
+If you are making contributions to our repositories on behalf of your company, then we will need a Corporate Contributor License Agreement (CLA) signed. In order to do that, please contact us at [contributions@strapi.io](mailto:contributions@strapi.io).
```

Comment:
`to our repositories` is it the wanted wording? (do we want to generalize for all the Strapi repositories here?)

Reply:
Yes, for every repository we need the CLA 👍